nano-layout.el

```diff
@@ -53,6 +53,7 @@
       initial-scratch-message nil)
 (tool-bar-mode 0)
 (tooltip-mode 0)
+(menu-bar-mode 0)
```

Comment:
This leads to flickers on OSX, this is the reason it was not included. I'll add the fix later.

Reply:
When you say you'll add the fix, do you mean you can merge this change "as is" and then fix it, or would you rather have me remove that line and take care of the rest ?

Reply:
You can leave. I'll got the flicker and then I will think about applyting the patch :)